cmd/controller/app/options/options.go

```diff
@@ -51,6 +53,8 @@ func (o *ControllerRunOptions) AddFlags(fs *pflag.FlagSet) {
 			"Enabling this will ensure there is only one active controller manager.")
 	fs.BoolVar(&o.EnableAddonDeploy, "enable-agent-deploy", o.EnableAddonDeploy,
 		"Enable deploy addon agent.")
+
+	fs.BoolVar(&o.EnableClusterClaims, "enable-clusterclaims", o.EnableClusterClaims, "Enable to create clusterclaims on the managed cluster")
```

Comment:
do we want to call it enableClusterClaim or enableHostedMode? I think it is not just disable the clusterclaim, for reading core/socket, are those part still working in hosted mode?

Reply:
Sure, let me check "reading core/socket" part of the code.

Reply:
`name: cluster1`

to

`name: {"hub1": "cluster1", "hub2": "cluster2"}`